src/apps/reservation-list/modal/reservation-details/reservation-details-buttons.tsx

```diff
@@ -6,25 +6,33 @@ import { MaterialProps } from "../../../loan-list/materials/utils/material-fetch
 export interface ReservationDetailsButtonProps {
   reservationId: number;
   numberInQueue?: number | null;
+  classNames?: string;
+  buttonClassNames?: string;
   openReservationDeleteModal: (deleteId: string) => void;
 }
 
 const ReservationDetailsButton: FC<
   ReservationDetailsButtonProps & MaterialProps
-> = ({ reservationId, numberInQueue, openReservationDeleteModal }) => {
+> = ({
+  reservationId,
+  numberInQueue,
+  openReservationDeleteModal,
+  classNames,
+  buttonClassNames
+}) => {
   const t = useText();
 
   return (
-    <div className="modal-details__buttons">
+    <div className={`modal-details__buttons ${classNames}`}>
       {numberInQueue && numberInQueue > 0 && (
-        <div className="my-8 mx-16 text-body-medium-regular">
+        <div className="my-8 mr-16 text-body-medium-regular">
           {t("reservationDetailsOthersInQueueText")}
         </div>
       )}
       <Button
         label={t("reservationDetailsButtonRemoveText")}
         onClick={() => openReservationDeleteModal(reservationId.toString())}
-        classNames="btn-primary btn-filled btn-small arrow__hover--right-small"
+        classNames={`btn-primary btn-filled btn-small arrow__hover--right-small ${buttonClassNames}`}
```

Comment:
You do not have to specify all those classes. Instead you should use the props: size and variant.